src/utils/strings.cpp

```diff
@@ -424,5 +425,11 @@ std::string find_string_prefix(const std::string &input, char separator)
     }
     return input.substr(0, current);
 }
+
+bool has_space(const std::string &str)
+{
+    return std::any_of(str.begin(), str.end(), [](unsigned char c) { return std::isspace(c); });
```

Comment:
Have check the absl implementation is better than Cpp standard or not?

```
// Determines whether the given character is a whitespace character (space,
// tab, vertical tab, formfeed, linefeed, or carriage return).
inline bool ascii_isspace(unsigned char c) {
  return (ascii_internal::kPropertyBits[c] & 0x08) != 0;
}
```

Reply:
OK, once this function could be used, no need to use lambda expression to wrap `std::isspace`. I'll try it.